classes/mender-uboot.bbclass

```diff
@@ -7,6 +7,3 @@ IMAGE_BOOT_ENV_FILE ?= "uboot.env"
 IMAGE_BOOT_FILES_append = " ${IMAGE_BOOT_ENV_FILE}"
 
 EXTRA_IMAGEDEPENDS += "u-boot"
-
-DEPENDS_mender_append = " u-boot u-boot-fw-utils"
-RDEPENDS_mender_append = " u-boot u-boot-fw-utils"
```

Comment:
NACK. There needs to be a way to turn off the U-Boot dependency for Mender. I don't think it's working fully as it should with the `mender-uboot` class; it will need more fixing and perhaps switching to [MEN-595](https://tracker.mender.io/browse/MEN-595); but putting it here would bring us even further from that goal.

In the future, we will allow Mender to update boot loader properties by calling user supplied commands, and in this case there is no dependency on U-Boot.

What is the warning that bitbake raises regarding this?